src/dune_pkg/better_candidate.mli

```diff
@@ -0,0 +1,21 @@
+open Import
+
+type result =
+  | Better_candidate of { version : OpamPackage.Version.t }
+  | Package_not_found
+  | Package_is_best_candidate
+
+type t =
+  { is_immediate_dep_of_local_package : bool
+  ; result : result
+  ; pkg : Lock_dir.Pkg.t
+  }
+
+val outdated_packages : t list -> (bool * Import.Package_name.t * string * string) list
+val packages_that_were_not_found : t list -> Import.Package_name.t list
+
+val outdated_status
+  :  transitive:bool
+  -> Path.Source.t
```

Comment:
It's not clear what the `Path.Source.t` argument is

Reply:
It's the lock directory. I've cleared this up now.